### PR TITLE
Add travis.

### PR DESCRIPTION
**Pull request by robdodson**
Check that PRs don't break the build command.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "10.14.1"
```
Comment (ebidel) [inline .travis.yml]: ">= 10"?

Comment (ebidel) [reply .travis.yml]: To match package.json

Comment (robdodson) [reply .travis.yml]: Sam had a thing that only worked on node v10.11 for a bit... let's see if that was fixed. I'll try v10

Comment (robdodson) [reply .travis.yml]: using "10" seems to work. I don't think travis supports >= so i'll just go with 10 for now.

**Pull request merged**